tensorboard/backend/application.py

```diff
@@ -263,7 +264,25 @@ def __init__(self, plugins, path_prefix=''):
         else:
           path = (self._path_prefix + DATA_PREFIX + PLUGIN_PREFIX + '/' +
                   plugin.plugin_name + route)
-        self.data_applications[path] = app
+
+        if path.endswith('/*'):
+          # Note we remove the '*' but leave the slash in place.
+          path = path[:-1]
+          if '*' in path:
+            raise ValueError('Invalid route: only trailing wildcards are supported (i.e., `/.../*`)')
```

Comment:
Please include the text and plugin name in the error message. (Note that
in the `if`-case you’ll have to re-append the asterisk, or equivalent.)

Reply:
done